vscode/src/toitExec.ts

```diff
@@ -36,12 +40,20 @@ async function executeRunCommand(ctx: Context, device?: Device) {
 
   if (!device) return;  // Device selection prompt dismissed.
 
-  try {
-    ctx.output.startDeviceOutput(device);
-    toitSpawn(ctx, "dev", "-d", device.name, "run", filePath);
+  try {    
+    const fileName = basename(filePath);
+    const out = ctx.output.deviceOutput(device);
+    out.show();
+    const cp = toitExecFile(ctx, "dev", "-d", device.name, "run", filePath);
+    cp.stderr?.on("data", (message: any) => {
+      out.send(fileName, message);
+    });
+    cp.stdout?.on("data", (message: any) => {
+      out.send(fileName, message);
+    });
```

Comment:
I don't think this guarantees that you get individual lines.
The stdout and stderr could be completely mixed up.
Unless we want to show stdout and stderr differently (which is not the case here), it would be better to give the subprocess the same pipe for stdout and stderr.
Not sure if that's easily possible, but most frameworks make it possible to specify how stdout/stderr/stdin are set up.

Reply:
I don't think it is possible (for reference see https://nodejs.org/api/child_process.html#child_process_child_process_execfile_file_args_options_callback).

We can change it to use await and print the `stdout` and `stderr` separately, but then we would have to wait until the program is done executing. I like this solution best of the ones, I have tried.